qemu/tests/win_virtio_driver_installer_repair.py

```diff
@@ -18,7 +18,7 @@ def run(test, params, env):
     3) Boot guest with all virtio device.
     4) Install driver via virtio-win-guest-tools.exe.
     5) Run virtio-win-guest-tools.exe signature check command in guest.
-    6) Run QEMU FWCfg Device installed check command in guest.
+    6) Run QEMU FwCfg Device installed check command in guest.
```

Comment:
This step is not special anymore so that we can remove it directly. (remember to sync the polarion case)

Reply:
Yes,you are right.Updated the code.